### PR TITLE
Fix load/store links in rvv-intrinsic-api.md

### PR DESCRIPTION
**Pull request by topperc**
Signed-off-by: Craig Topper <craig.topper@sifive.com>

### CODE REVIEW

**Comment by hdelassus** [comment]
LGTM. I think there's a few more broken anchors

**Review by eopXD — COMMENTED** [review]
Looks good except the comments.

File: intrinsic_funcs/08_vector_mask_functions.md [hunk 1]
```diff
@@ -309,4 +309,4 @@ vuint64m1_t vid_v_u64m1_m (vbool64_t mask, vuint64m1_t maskedoff, size_t vl);
 vuint64m2_t vid_v_u64m2_m (vbool32_t mask, vuint64m2_t maskedoff, size_t vl);
 vuint64m4_t vid_v_u64m4_m (vbool16_t mask, vuint64m4_t maskedoff, size_t vl);
 vuint64m8_t vid_v_u64m8_m (vbool8_t mask, vuint64m8_t maskedoff, size_t vl);
-```
\ No newline at end of file
+```
```
[the remark this answers is not in the document]
Comment (hdelassus) [reply intrinsic_funcs/08_vector_mask_functions.md:312]: This is an automatic edit making sure the file ends with a conventional newline.

File: intrinsic_funcs.md [hunk 2]
```diff
@@ -22823,7 +22823,7 @@ vfloat64m1_t vfwredusum_vs_f32m8_f64m1_m (vbool4_t mask, vfloat64m1_t dest, vflo
 ```
 ## Vector Mask Functions:
 
-### [Vector Mask Load/Store Functions](../rvv-intrinsic-api.md#74-vector-mask-load-operations):
+### [Vector Mask Load/Store Functions](../rvv-intrinsic-api.md#74-vector-unit-stride-operations):
```
[the remark this answers is not in the document]
Comment (hdelassus) [reply intrinsic_funcs.md:22826]: The problem is that there is no such anchor. Mask load/store ops and unit-stride load/store ops are regrouped under [Vector unit-stride ops](https://github.com/riscv-non-isa/rvv-intrinsic-doc/blob/master/rvv-intrinsic-api.md#74-vector-unit-stride-operations)

**Review by eopXD — APPROVED** [review]
Thank you for the explanation @hdelassus, the PR looks good to me now.

**Pull request merged**